himanshu/postgresAssignment/src/routes/resultRoutes.ts

```diff
@@ -0,0 +1,13 @@
+export {};
+const Koaroute = require('@koa/router');
+
+const router = new Koaroute();
+const resultController = require('../controllers/resultContoller.ts');
+
+router.post('/marks', resultController.addMarks);
+router.put('/marks', resultController.updateMarks);
+router.get('/marks/:studentId', resultController.getMarksByStudentId);
+router.get('/highestmarkspersubject/:classId', resultController.getHighestMarksPerSubject);
```

Comment:
/marks/subject/:subjectIdId/class/:classId or we can use query param. let use the first option for now, check how we can do with query param also if you get free time.